lib/modules/versioning/cargo/index.spec.ts

```diff
@@ -121,7 +129,11 @@ describe('modules/versioning/cargo/index', () => {
     ${'5.0'}                 | ${'bump'}     | ${'5.0.0'}     | ${'5.0.7'}      | ${'5.0'}
     ${'5.0'}                 | ${'bump'}     | ${'5.0.0'}     | ${'5.1.7'}      | ${'5.1'}
     ${'5.0'}                 | ${'bump'}     | ${'5.0.0'}     | ${'6.1.7'}      | ${'6.1'}
-    ${'5.0'}                 | ${'replace'}  | ${'5.0.0'}     | ${'6.1.7'}      | ${'6.1'}
+    ${'0.5'}                 | ${'bump'}     | ${'0.5.0'}     | ${'0.5.1'}      | ${'0.5'}
+    ${'0.5'}                 | ${'bump'}     | ${'0.5.0'}     | ${'0.6.1'}      | ${'0.6'}
+    ${'5.0'}                 | ${'replace'}  | ${'5.0.0'}     | ${'5.1.7'}      | ${'5.0'}
+    ${'5.0'}                 | ${'replace'}  | ${'5.0.0'}     | ${'6.1.7'}      | ${'6.0'}
```

Comment:
Is `5.0` logically the same as `5`? If so, is there any reason why someone would choose `5.0` instead of `5`?

Reply:
For 5.0 particularly, yes, it's the same as 5.

However, you can always write a 5.1 there, indicating that you have probably used something exclusively >=5.1, and it should remain 5.1 that way (not 5, and not e.g. 5.1.2).

In the test case this distinction does not matter, but if you find it misleading in some way, I can always add some more test cases for 5.1.

Reply:
I was more thinking about whether a user would expect 6.0 here or 6.1 - because it's now changed in this PR compared to before.

Here's what I think:
 - if rangeStrategy=bump then it should go to 6.1
 - if rangeStrategy=replace then it should go to 6.0

Can you add a test for the former if it's not already there?

Reply:
@rarkins The test is already there, so I'll go ahead and mark this conversation as resolved.